nilearn/surface/surface_image.py

```diff
@@ -4,97 +4,100 @@
 import abc
 import dataclasses
 import pathlib
-from typing import Dict
 
-import numpy as np
+from nilearn.surface import _io
 
-from nilearn.experimental.surface import _io
 
-PolyData = Dict[str, np.ndarray]
-
-
-class Mesh(abc.ABC):
+class _Mesh(abc.ABC):
     """A surface :term:`mesh` having vertex, \
-    coordinates and faces (triangles)."""
+    coordinates and :term:`faces` (triangles)."""
 
-    n_vertices: int
+    n_vertices: ...
```

Comment:
OK I am sorry but I am putting types back in here because `...` is just worse than mentioning the type.

Reply:
Fair. I wasn't sure what to do about these so it's a bit unfinished. I think you should add back PolyMesh and PolyData as well

Reply:
No worries. I was just confused as to why we were "ellisping" here.

In general, still sad we are removing type hints that were already here though.

Reply:
you could remove the whole line or leave the type.

> In general, still sad we are removing type hints that were already here though.

if they are complete enough within a sub-package that this part can successfully pass the mypy check we could leave them. what I would like to avoid is broken type hints, which is bound to happen if we don't run mypy in the ci